src/components/Home.js

```diff
@@ -1,14 +1,35 @@
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../apis/axiosClient';
+import AlbumsList from './AlbulList';
 
 function Home() {
   const { status, user } = useAuth();
 
+  // testing axiosInstance request
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    axiosInstance
+      .get('/showMe')
+      .then((response) => {
+        setData(response.data);
+        console.log('this is axios instance' + data);
+      })
+      .catch((error) => {
+        console.error('Error fetching data:', error);
+      });
+  }, []);
+
+  // end of testing
+
   return (
-    <div>
+    <>
       <h1>THIS IS THE HOME PAGE</h1>
       <p>Current log-in status is: {status}</p>
       <p>Current logged-in user is: {JSON.stringify(user)}</p>
```

Comment:
We can probably remove this content at this point since we now have actually something to show here?

Reply:
Removed